build/gulp/plugins/util/getComponentInfo.ts

```diff
@@ -99,14 +99,21 @@ const getComponentInfo = (filepath: string) => {
     const { description, tags } = parseDocblock(propDef.description)
     const { name, value } = parseType(propName, propDef)
 
-    info.props[propName] = {
-      ...propDef,
-      description,
-      tags,
-      value,
-      defaultValue: parseDefaultValue(propDef),
-      name: propName,
-      type: name,
+    const parentInterface = _.get(propDef, 'parent.name')
+    const ignoredInDefinitions = _.includes(ignoredParentInterfaces, parentInterface)
+
+    if (!ignoredInDefinitions) {
```

Comment:
nit: would suggest to invert condition and switch `if` and `else` branches, it will be easier to read

Reply:
Done 👍